### PR TITLE
Usdz exporter

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15707/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15707/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15707/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15707/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15707/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15707/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15707/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15707/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15707/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15707/merge/testResults/webgl2playwright/index.html

File: packages/dev/serializers/src/USDZ/usdzExporter.ts [hunk 1]
```diff
@@ -0,0 +1,673 @@
...
+import { Constants } from "core/Engines/constants";
+import type { Material } from "core/Materials/material";
+import type { PBRMaterial } from "core/Materials/PBR/pbrMaterial";
+import type { PBRMetallicRoughnessMaterial } from "core/Materials/PBR/pbrMetallicRoughnessMaterial";
+import type { StandardMaterial } from "core/Materials/standardMaterial";
+import type { BaseTexture } from "core/Materials/Textures/baseTexture";
+import type { Texture } from "core/Materials/Textures/texture";
+import { Color3 } from "core/Maths/math.color";
+import { Matrix, Vector2 } from "core/Maths/math.vector";
+import type { Geometry } from "core/Meshes";
```
Comment (Popov72) [inline packages/dev/serializers/src/USDZ/usdzExporter.ts:12]:
```suggestion
import type { Geometry } from "core/Meshes/geometry";
```

**Review by Popov72 — CHANGES_REQUESTED** [review]
There's no need to export camera(s)?

**Comment by deltakosh** [comment]
There is but in a second pr

**Review by Popov72 — APPROVED** [review]

**Pull request merged**